django/views/generic/edit.py

```diff
@@ -39,7 +39,7 @@ def get_form_kwargs(self):
             'prefix': self.get_prefix(),
         }
 
-        if self.request.method in ('POST', 'PUT'):
+        if self.request.method in ('POST', 'PUT', 'DELETE'):
```

Comment:
I think this change has to be out of scope for this PR. (I want to use a success message with `DeleteView`...)

Reply:
Agreed. Removed.